modules/moordyn/src/MoorDyn_Driver.f90

```diff
@@ -132,7 +132,7 @@ PROGRAM MoorDyn_Driver
    IF ( LEN( TRIM(FlagArg) ) > 0 ) CALL NormStop()
 
       ! Display the copyright notice
-   CALL DispCopyrightLicense( version%Name, 'Copyright (C) 2021 NREL, 2019 Matt Hall' )
+   CALL DispCopyrightLicense( version%Name, 'Copyright (C) 2024 NREL, 2019 Matt Hall' )
```

Comment:
You could get rid of the '2024 NREL' copyright here since that's done automatically in `DispCopyrightLicense`.

Reply:
Resolved with my most recent commit